e2e/nomostest/new.go

```diff
@@ -275,6 +276,12 @@ func FreshTestEnv(t nomostesting.NTB, opts *ntopts.New) *NT {
 	// Speed up the delay between rendering attempts to speed up testing (default: 5s)
 	nt.HydrationPollingPeriod = 1 * time.Second
 
+	if nt.IsGKEAutopilot {
+		nt.DefaultWaitTimeout = 10 * time.Minute
```

Comment:
nt.IsGKEAutopilot is set after `nt.detectGKEAutopilot`, it has to be after.

Reply:
Great catch, thank you! Fixed